core/pom.xml

```diff
@@ -140,45 +127,34 @@
         <!-- Lucene spatial -->
 
 
-        <!-- START: dependencies that are shaded -->
+        <!-- START: dependencies that might be shaded -->
```

Comment:
oh I see... I understood the comment as "some of them are shaded, others aren't"